sencha-workspace/SlateDemonstrationsStudent/app/controller/Dashboard.js

```diff
@@ -22,7 +22,8 @@ Ext.define('SlateDemonstrationsStudent.controller.Dashboard', {
    // controller configuration
     views: [
         'SlateDemonstrationsStudent.view.CompetencyCard',
-        'Slate.cbl.view.student.skill.OverviewWindow'
+        'SlateDemonstrationsStudent.view.OverviewWindow',
+        'SlateDemonstrationsStudent.view.OverviewWindowController'
```

Comment:
@aliyeysides you should not be requiring the ViewController directly, the view is what requires and binds to its viewcontroller, your app controller should just be requiring the view. I originally noted this here: https://github.com/SlateFoundation/slate-cbl/pull/132#discussion_r65588891

Also, one of the benefits of using the specialized `views` config in app controllers to require things is that it knows to automatically prepend `AppName.view.` if you just provide it a class name, so this should just be:

Finally, as I noted here this is not the place where CompetencyCard should be getting required: https://github.com/SlateFoundation/slate-cbl/pull/132#discussion-diff-65590087

```
views: [
    'OverviewWindow'
]
```